examples/02-plot/lookup-table.py

```diff
@@ -0,0 +1,236 @@
+"""
+.. _lookup_table_example:
+
+Anti-Aliasing
+~~~~~~~~~~~~~
+Demonstrate the usage of a lookup table within PyVista
+
+The :class:`pyvista.LookupTable` can be used to have fine-tuned control over
+the mapping between a :class:`pyvista.DataSet`'s scalars and RGBA colors.
+
+"""
+
+import pyvista as pv
+from pyvista import examples
+
+pv.set_plot_theme('document')
```

Comment:
Isn't this the default? It should be and we shouldn't need this IMO

Reply:
```suggestion
```